### ISSUE TITLE
Task - Image matrix - cannot move image from one cell to another.

### ISSUE THREAD

**Issue by proceps** [SpeciesFileGroup/taxonworks#2874]

**Comment by mjy** [comment]
There is an array/array comparison rais in the admin emails that is likely related.

**Comment by jlpereira** [comment]
I can't replicate this, but I pushed some changes for cell component that maybe could fix it.

**Comment by proceps** [comment]
![image](https://user-images.githubusercontent.com/5597748/161447679-13009e5c-b081-4731-84bd-b3f3a5f761a4.png)

In the image matrix, when I drag an image from one cell to another, I have an error message. The image sems to move, but if you reload the page, the image stays in the original cell.
To experiment. I used 3i database in sandstorm. I the matrix dashboard I searched for the genus Hymetta, selected all species with scores, Opened the matrix in the view mode (openning the matrix in edit mode directly does not work now the matrix for images is not visible in the list). Should be another issue.
Once I load the matrix, I can switch to edit mode and try to drag the images.
